### DISCUSSION TITLE
Thunderbird update policy

### DISCUSSION THREAD

**Discussion by woj-tek** [Homebrew/discussions#5513] [Casks]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

What is the update policy regarding Casks? Thunderbird was release a couple of days ago though they don't push auto-update to the users just yet (and it will rolled update I guess). I see that the cask (https://github.com/Homebrew/homebrew-cask/blob/master/Casks/t/thunderbird.rb) sits as last version ("115.13.0") just like the auto-update.

On the other hand there is the issue (https://github.com/Homebrew/homebrew-cask/issues/179792) and PR (https://github.com/Homebrew/homebrew-cask/pull/180036) aiming to update the cask (I guess) but the "livecheck" update hasn't been run on the cask itself (???).

So... does Homebrew tries to mimic auto-update behaviour or updates just as the binaries are released?

### Relevant casks

thunderbird

**Comment by SMillerDev** [comment]
> So... does Homebrew tries to mimic auto-update behaviour or updates just as the binaries are released?

It will update whenever the community makes a PR to update it. Or, in case of the autobump system, whenever livecheck detects there is a new version.

**Reply by woj-tek** [reply]
> Or, in case of the autobump system

Hmm... that was my initial assumption from the looks of the issues though I wasn't sure about how autobump works / is it automatic or if someone needs to manually run in and create PR :-)
